packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -461,6 +461,9 @@ module.exports = function (log, config, bounces) {
     try {
       await bounces.check(to, template);
     } catch (err) {
+      const tags = { template, error: err.errno };
+      statsd.increment('email.bounce', tags);
```

Comment:
We should name this metric differently.  This sounds like we are counting bounces.  I'm not good at naming but something that tells readers that emails were blocked from sending.